Sketch (1).java

```diff
@@ -0,0 +1,76 @@
+import processing.core.PApplet;
+
+public class Sketch extends PApplet {
```

Comment:
your filename is `Sketch(1).java` and your class is `Sketch`. The filename and the class name should be the same.  Also, you have two Sketch class definitions in your project.  This can cause compilation issues.